services/QuillLMS/engines/evidence/app/models/evidence/highlight.rb

```diff
@@ -8,10 +8,15 @@ class Highlight < ApplicationRecord
     
     MIN_TEXT_LENGTH = 1
     MAX_TEXT_LENGTH = 5000
-    TYPES= [
-      'passage',
-      'response',
-      'prompt'
+
+    TYPE_PASSAGE  = 'passage'
```

Comment:
This re-org allows use to reference Highlight constants from PrefilterCheck, for data integrity.

Reply:
Addresed.